app/models/miq_worker/runner.rb

```diff
@@ -487,7 +487,7 @@ def set_process_title
   def worker_options
     {
       :messaging => MiqQueue.messaging_client_options,
-      :settings  => Settings.to_hash
+      :settings  => worker_settings.to_hash
```

Comment:
`worker_settings` have strings like `"2.minutes"` converted to `120` automatically already